python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceConversationalRemoteInferencePromptSchema.ts

```diff
@@ -18,6 +18,46 @@ export const HuggingFaceConversationalRemoteInferencePromptSchema: PromptSchema
         to a deployed Inference Endpoint`,
         default: "facebook/blenderbot-400M-distill",
       },
+      parameters: {
+        type: "object",
+        description: "Additional parameters for the conversational task.",
+        properties: {
+          min_length: {
+            type: "integer",
+            description: "Integer to define the minimum length in tokens of the output summary..",
+          },
+          max_length: {
+            type: "integer",
+            description: "Integer to define the maximum length in tokens of the output summary."
+          },
```

Comment:
Is the default for this 20 like text generation? If so, let's set a default: 100 or something like what we have for [TextGeneration](https://github.com/lastmile-ai/aiconfig/blob/4197cf3c9c5fb1c3b85bcd6a83bbcd8493c6fa73/python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceTextGenerationRemoteInferencePromptSchema.ts#L54)

Reply:
synced offline: default doesn't work as great for conversational support.  Per docs: Models tend to have shorter max_length.

Decided not to add a default.